host/capture_control.cpp

```
@@ -0,0 +1,288 @@
+{
+    std::lock_guard<std::mutex> lock(mutex_);
+    plugin_load_results_.push_back(request);
+    return OperationStatus{
+        true,
+        request.runtime_state,
+        "plugin load result recorded",
+        0U,
+    };
+}
+
+const std::vector<WorkerLog>& CaptureControlEndpoint::GetWorkerLogs() const
```

Cppcheck (2.21.0)

[style] The function 'GetWorkerLogs' is never used.

(unusedFunction)